src/pages/buyers-guide/category.js

```diff
@@ -0,0 +1,15 @@
+var React = require(`react`);
```

Comment:
These are just stubs for now. I have them more fleshed out in the other patch but it made sense to separate them.